src/items/external-blocks.md

```diff
@@ -211,6 +230,40 @@ extern {
 }
 ```
 
+Using this attribute with the `link_ordinal` attribute will result in a
+compiler error.
+
+### The `link_ordinal` attribute
+
```

Comment:
There is generally a template for defining an attribute that looks like:

```
The *`link_ordinal` attribute* …
```

The first sentence should briefly explain what the attribute does. I'm not familiar with this, but it might look something like this:

The *`link_ordinal` attribute* specifies the numeric ordinal of an item in an [`extern` block](external-blocks.md) to link against.

Reply:
Ok, rewrote this so that the first sentence explains what the attribute does, and then I follow up with an explanation of what an ordinal is.